include/RAJA/util/Operators.hpp

```diff
@@ -419,15 +419,20 @@ struct bit_xor : public detail::binary_function<Arg1, Arg2, Ret> {
   }
 };
 
-// comparison
-
+// comparison 
+/*!
+ Checks if the candidate (rhs) value is strictly less than
+ the current value (lhs); if so the candidate is returned. 
+ When this operator is used to cycle through an array 
+ this ensures that the location of the first min/max is kept.
+*/
```

Comment:
@davidbeckingsale , I added the comment under the comparison comment.

Reply:
Looks good, thanks!